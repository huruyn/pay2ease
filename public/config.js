window.API_URL = "https://zippy-retail-api-dev.azurewebsites.net";