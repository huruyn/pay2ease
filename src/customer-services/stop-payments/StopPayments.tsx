import {
  Row,
  Col,
  Card,
  Form,
  Alert,
  FormControl,
  Button,
} from 'react-bootstrap';
import CommonPageContainer from '../../common/CommonPageContainer';

const Divider = () => <div className="border-top my-3" />;

export default function StopPayments() {
  return (
    <div className="page-content">
      <CommonPageContainer title="Stop Payment">
        <Row>
          <Col md={9}>
            <Row className="mt-3">
              <Col>
                <b>Step: 1 - 2</b>
              </Col>
            </Row>
            <h4 className="mt-3 mb-2">Before you begin</h4>
            <p>
              Contact the merchant or payee to ccancel the pre-authorized debit or
              void the cheque. If the merchant or payee are unable to accommodate
              your request, you can complete this form.
            </p>
            <ul>
              <li>
                It&apos;s important that the payment details provided are accurate
                so we can successfully stop the payment on your behalf.
              </li>
              <ul>
                <li>
                  The request must be submitted by 6 pm at least 1 business day in
                  advance of when the payment is due.
                </li>
                <li>
                  Refer to your transaction history for specific payee name
                  details.
                </li>
                <li>
                  Verify the amount using your statement. If the amount is
                  unknown, leave the amount field blank
                </li>
                <li>
                  If the information isn&apos;t accurate, we&apos;ll do our best
                  to match with details provided, but we may not always be
                  successful.
                </li>
              </ul>
              <li>
                The fee of this service varies by the level of information
                provided.
              </li>
              <ul>
                <li>
                  $12.50: If you have complete payee and amount details and enter
                  them in the form below.
                </li>
                <li>
                  $20.00: If you have complete payee details and the amount of the
                  payment is unknown. Tip: use this option if the amount
                  isn&apos;t the same every month for the PAD.
                </li>
              </ul>
              <li>
                The request will be in effect for 12 months. After this point,
                contact the merchant or payee to stop additional payments or
                submit another request using this form.
              </li>
              <li>
                To confirm if the stop payment request has been successful, review
                your account transaction history in 2 business days. You may
                notice that the payment was prcessed and then returned.
              </li>
              <li>
                If you would like to stop a ZippyX payment, you can call at
                1-800-465-2422
              </li>
            </ul>
            <div>
              <h5
                className="pb-2 pt-2 mb-2 mt-3"
                style={{ borderBottom: '1px solid grey' }}
              >
                Step 1: Enter the account and payment details
              </h5>
              <p>Select the account you used to make the payment.</p>
              <Row className="pb-2 pt-2">
                <Col md={4}>
                  <Form.Label style={{ color: 'grey', fontSize: 14 }}>
                    From account:
                  </Form.Label>
                </Col>
                <Col md={6}>
                  <Form.Select>
                    <option>Choose an account</option>
                  </Form.Select>
                </Col>
              </Row>
              <Divider />
              <Row className="pb-2 pt-2">
                <Col md={12}>
                  <p>
                    Make sure the payee name is accurate and entered exxactly as
                    shown in your transaction history
                  </p>
                  <Alert variant="secondary">
                    Tip: You can copy the exact name of merchant or payee from
                    your transaction history. The payee name is after
                    &quot;PREAUTHORIZED DEBIT.&quot; For example, with
                    &quot;PREAUTHORIZED DEBIT SAMPLE LTD,&quot; the payee name
                    would be &quot;SAMPLE LTD&quot;.
                  </Alert>
                </Col>
                <Col md={4}>
                  <Form.Label style={{ color: 'grey', fontSize: 14 }}>
                    Payee name:
                  </Form.Label>
                </Col>
                <Col md={6}>
                  <FormControl type="text" />
                </Col>
              </Row>
              <Divider />
              <Row className="pb-2 pt-2">
                <Col md={4}>
                  <Form.Label style={{ color: 'grey', fontSize: 14 }}>
                    Payment amount:
                  </Form.Label>
                </Col>
                <Col md={6}>
                  <FormControl type="text" />
                  <Form.Check type="checkbox" label="I dont't know the amount" />
                </Col>
              </Row>
              <Divider />
            </div>
            <div>
              <h5
                className="pb-2 pt-2 mb-2 mt-3"
                style={{ borderBottom: '1px solid grey' }}
              >
                Step 2: Select the payment type:
              </h5>
              <Row className="pb-2 pt-2">
                <Col md={4}>
                  <Form.Label style={{ color: 'grey', fontSize: 14 }}>
                    Payment type:
                  </Form.Label>
                </Col>
                <Col md={6}>
                  <Form.Check type="radio" label="Stop a pre-authorized debit" />
                  <Form.Check type="radio" label="Stop a payment on a cheque" />
                </Col>
              </Row>
              <Divider />
              <Row className="pb-2 pt-2">
                <Col md={4}>
                  <Form.Label style={{ color: 'grey', fontSize: 14 }}>
                    Cheque number(3 digits):
                  </Form.Label>
                </Col>
                <Col md={6}>
                  <FormControl type="text" />
                </Col>
                <Alert variant="secondary" className="mt-3">
                  Tip: The cheque number is the first 3 digits of the code on the
                  bottom-left corner of the cheque.
                </Alert>
              </Row>
              <Divider />
              <Row className="pb-2 pt-2">
                <Col md={4}>
                  <Form.Label style={{ color: 'grey', fontSize: 14 }}>
                    Date on cheque:
                  </Form.Label>
                </Col>
                <Col md={8}>
                  <Row>
                    <Col md={4}>
                      <FormControl placeholder="Month" type="text" />
                    </Col>
                    <Col md={3}>
                      <FormControl placeholder="Day" type="text" />
                    </Col>
                    <Col md={3}>
                      <FormControl placeholder="Year" type="text" />
                    </Col>
                    <Col md={2}>
                      <div
                        style={{
                          width: 20,
                          height: 25,
                          border: '1px dotted grey',
                          textAlign: 'center',
                          marginRight: 10,
                        }}
                      >
                        P
                      </div>
                    </Col>
                  </Row>
                </Col>
              </Row>
              <Divider />
            </div>
            <div>
              <h5
                className="pb-2 pt-2 mb-2 mt-3"
                style={{ borderBottom: '1px solid grey' }}
              >
                Step 3: Choose an account to pay the stop payment fee
              </h5>
              <p className="mt-2">
                Select the account from which to pay the fee.
              </p>
              <Row className="pb-2 pt-2 mt-2">
                <Col md={4}>
                  <Form.Label style={{ color: 'grey', fontSize: 14 }}>
                    Service fee from account:
                  </Form.Label>
                </Col>
                <Col md={6}>
                  <Form.Select>
                    <option>Choose an account</option>
                  </Form.Select>
                </Col>
              </Row>
            </div>
            <div>
              <h5
                className="pb-2 pt-2 mb-2 mt-3"
                style={{ borderBottom: '1px solid grey' }}
              >
                Step 4: Review the details and confirm that you want to proceed
              </h5>
              <p>
                Read the following Terms and Conditions and select &quot;I
                agree&quot; to request stop payment.
              </p>
              <Row className="mt-3">
                <Col md="12">
                  <div className="p-4" style={{ border: '1px solid grey' }}>
                    <h4>Stop Payment Terms and Conditions</h4>
                    <p>
                      (Not applicable to stop payments on pre-authorized payments
                      set up using ZippyX Advantage Debit Card
                      <sup>TM1</sup>
                      details
                      <sup>1</sup>
                      )
                    </p>
                    <ul>
                      <li>
                        ZippyX will attempt to stop payment of the cheque or
                        pre-authorized debit described below(&quot;item&quot;) and will
                        require a reasonable period to process your stop payment
                        order (&quot;order&quot;).
                      </li>
                      <li>
                        If your order cannot be processed before the item is paid
                        to the payee, we will refund the service charge and have
                        not further liability to you. An order received after 6 pm
                        ET will be considered to be received the next banking day.
                      </li>
                      <li>
                        Your order is in effect for 12 months from the date it is
                        submitted, or the date of the item whichever is later.
                      </li>
                    </ul>
                  </div>
                  <div
                    className="p-4"
                    style={{ background: '#a1a1a1', color: 'white' }}
                  >
                    <Form.Check
                      type="checkbox"
                      name="agree-terms-conditions"
                      label="I agree to these terms and conditions and wish to request a stop payment."
                      id="agree-terms-conditions"
                    />
                  </div>
                </Col>
              </Row>
            </div>
            <div
              className="mt-4"
              style={{
                borderTop: '1px solid grey',
                borderBottom: '1px solid grey',
              }}
            >
              <Row className="mt-4 mb-4">
                <Col>
                  <Button variant="light" className="d-flex">
                    <div
                      style={{
                        width: 20,
                        height: 25,
                        border: '1px dotted grey',
                        textAlign: 'center',
                        marginRight: 10,
                      }}
                    >
                      P
                    </div>
                    Cancel
                  </Button>
                </Col>
                <Col className="d-flex justify-content-end">
                  <Button className="zippy-btn d-flex">
                    Next
                  </Button>
                </Col>
              </Row>
            </div>
            <div className="mt-2 pt-2">
              <div className="d-flex">
                <div
                  style={{
                    width: 20,
                    height: 25,
                    border: '1px dotted grey',
                    textAlign: 'center',
                    marginRight: 10,
                  }}
                >
                  P
                </div>
                <span style={{ fontWeight: 'bold' }}>Note:</span>
              </div>
              <p className="mt-2">
                <b>Service Fee Details</b>
              </p>
              <p>
                A stop payment fee of $12.50 will be charged when you provide full
                details.
              </p>
              <p>
                A stop payment fee of $20 will be charged when you don&apos;t
                provide full details
              </p>
              <p>Full details on a cheque are Cheque Number and Amount.</p>
              <p>
                Full details on a pre-authorized payment are Payee and Amount.
              </p>
              <p>
                This fee doesn&apos;t apply to ZippyX Smart
                <sup>TM</sup>
                Plus Accounts or Wood Gundy Asset Advantage Accounts.
              </p>
              <p>
                <sup>1</sup>
                For stop payments on pre-authorized debits or recurring payments
                set up with ZippyX Advantage Debit Card details, contact the
                merchant, biller or payee directly and tell them to stop charging
                your account through your ZippyX Advantage Debit Card. If they
                don&apos;t stop, ZippyX may be able to work with Visa* to request a
                refund for you.
              </p>
              <p>
                <sup>TM 1</sup>
                Trandemark of ZippyX.
              </p>
              <p>
                <sup>*</sup>
                Trandemark of Visa Int., used under license
              </p>
            </div>
          </Col>
          <Col md={3}>
            <Card className="p-3 bg-white mt-2">
              <Card.Title>You can also:</Card.Title>
              <Card.Body>
                <a
                  href="/"
                  className="text-decoration-none text-black"
                  style={{
                    fontSize: 12,
                    marginTop: 5,
                    marginBottom: 5,
                    display: 'block',
                  }}
                >
                  Pay Bills
                  <span style={{ color: 'rgb(220, 53, 69)', marginLeft: 5 }}>
                    {'>'}
                  </span>
                </a>
              </Card.Body>
            </Card>
          </Col>
        </Row>
      </CommonPageContainer>
    </div>
  );
}
