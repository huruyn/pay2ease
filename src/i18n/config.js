import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

i18n.use(initReactI18next).init({
  fallbackLng: 'en-US',
  lng: 'en-US',
  resources: {
    'en-CA': {
      translations: require('./locales/en-CA/translations.json')
    },
    'en-US': {
      translations: require('./locales/en-US/translations.json')
    },
    'es-US': {
      translations: require('./locales/es-US/translations.json')
    },
    'fr-CA': {
      translations: require('./locales/fr-CA/translations.json')
    },
  },
  ns: ['translations'],
  defaultNS: 'translations'
});

i18n.languages = ['en-CA', 'en-US', 'es-US', 'fr-CA'];

export default i18n;