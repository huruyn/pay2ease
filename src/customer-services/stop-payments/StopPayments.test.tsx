import '../../i18n/config';
import StopPayments from './StopPayments';

const ReactTestRenderer = require('react-test-renderer');

describe('StopPayments Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(<StopPayments />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
