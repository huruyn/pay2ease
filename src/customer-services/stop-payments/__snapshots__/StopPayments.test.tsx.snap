// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StopPayments Component matches the snapshot 1`] = `
<div
  className="page-content"
>
  <div
    className="d-flex justify-content-between align-items-center"
  >
    <h3>
      Stop Payment
    </h3>
  </div>
  <div
    className="content-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-md-9"
      >
        <div
          className="mt-3 row"
        >
          <div
            className="col"
          >
            <b>
              Step: 1 - 2
            </b>
          </div>
        </div>
        <h4
          className="mt-3 mb-2"
        >
          Before you begin
        </h4>
        <p>
          Contact the merchant or payee to ccancel the pre-authorized debit or void the cheque. If the merchant or payee are unable to accommodate your request, you can complete this form.
        </p>
        <ul>
          <li>
            It's important that the payment details provided are accurate so we can successfully stop the payment on your behalf.
          </li>
          <ul>
            <li>
              The request must be submitted by 6 pm at least 1 business day in advance of when the payment is due.
            </li>
            <li>
              Refer to your transaction history for specific payee name details.
            </li>
            <li>
              Verify the amount using your statement. If the amount is unknown, leave the amount field blank
            </li>
            <li>
              If the information isn't accurate, we'll do our best to match with details provided, but we may not always be successful.
            </li>
          </ul>
          <li>
            The fee of this service varies by the level of information provided.
          </li>
          <ul>
            <li>
              $12.50: If you have complete payee and amount details and enter them in the form below.
            </li>
            <li>
              $20.00: If you have complete payee details and the amount of the payment is unknown. Tip: use this option if the amount isn't the same every month for the PAD.
            </li>
          </ul>
          <li>
            The request will be in effect for 12 months. After this point, contact the merchant or payee to stop additional payments or submit another request using this form.
          </li>
          <li>
            To confirm if the stop payment request has been successful, review your account transaction history in 2 business days. You may notice that the payment was prcessed and then returned.
          </li>
          <li>
            If you would like to stop a ZippyX payment, you can call at 1-800-465-2422
          </li>
        </ul>
        <div>
          <h5
            className="pb-2 pt-2 mb-2 mt-3"
            style={
              Object {
                "borderBottom": "1px solid grey",
              }
            }
          >
            Step 1: Enter the account and payment details
          </h5>
          <p>
            Select the account you used to make the payment.
          </p>
          <div
            className="pb-2 pt-2 row"
          >
            <div
              className="col-md-4"
            >
              <label
                className="form-label"
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 14,
                  }
                }
              >
                From account:
              </label>
            </div>
            <div
              className="col-md-6"
            >
              <select
                className="form-select"
              >
                <option>
                  Choose an account
                </option>
              </select>
            </div>
          </div>
          <div
            className="border-top my-3"
          />
          <div
            className="pb-2 pt-2 row"
          >
            <div
              className="col-md-12"
            >
              <p>
                Make sure the payee name is accurate and entered exxactly as shown in your transaction history
              </p>
              <div
                className="fade alert alert-secondary show"
                role="alert"
              >
                Tip: You can copy the exact name of merchant or payee from your transaction history. The payee name is after "PREAUTHORIZED DEBIT." For example, with "PREAUTHORIZED DEBIT SAMPLE LTD," the payee name would be "SAMPLE LTD".
              </div>
            </div>
            <div
              className="col-md-4"
            >
              <label
                className="form-label"
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 14,
                  }
                }
              >
                Payee name:
              </label>
            </div>
            <div
              className="col-md-6"
            >
              <input
                className="form-control"
                type="text"
              />
            </div>
          </div>
          <div
            className="border-top my-3"
          />
          <div
            className="pb-2 pt-2 row"
          >
            <div
              className="col-md-4"
            >
              <label
                className="form-label"
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 14,
                  }
                }
              >
                Payment amount:
              </label>
            </div>
            <div
              className="col-md-6"
            >
              <input
                className="form-control"
                type="text"
              />
              <div
                className="form-check"
              >
                <input
                  className="form-check-input"
                  disabled={false}
                  type="checkbox"
                />
                <label
                  className="form-check-label"
                  title=""
                >
                  I dont't know the amount
                </label>
              </div>
            </div>
          </div>
          <div
            className="border-top my-3"
          />
        </div>
        <div>
          <h5
            className="pb-2 pt-2 mb-2 mt-3"
            style={
              Object {
                "borderBottom": "1px solid grey",
              }
            }
          >
            Step 2: Select the payment type:
          </h5>
          <div
            className="pb-2 pt-2 row"
          >
            <div
              className="col-md-4"
            >
              <label
                className="form-label"
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 14,
                  }
                }
              >
                Payment type:
              </label>
            </div>
            <div
              className="col-md-6"
            >
              <div
                className="form-check"
              >
                <input
                  className="form-check-input"
                  disabled={false}
                  type="radio"
                />
                <label
                  className="form-check-label"
                  title=""
                >
                  Stop a pre-authorized debit
                </label>
              </div>
              <div
                className="form-check"
              >
                <input
                  className="form-check-input"
                  disabled={false}
                  type="radio"
                />
                <label
                  className="form-check-label"
                  title=""
                >
                  Stop a payment on a cheque
                </label>
              </div>
            </div>
          </div>
          <div
            className="border-top my-3"
          />
          <div
            className="pb-2 pt-2 row"
          >
            <div
              className="col-md-4"
            >
              <label
                className="form-label"
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 14,
                  }
                }
              >
                Cheque number(3 digits):
              </label>
            </div>
            <div
              className="col-md-6"
            >
              <input
                className="form-control"
                type="text"
              />
            </div>
            <div
              className="fade mt-3 alert alert-secondary show"
              role="alert"
            >
              Tip: The cheque number is the first 3 digits of the code on the bottom-left corner of the cheque.
            </div>
          </div>
          <div
            className="border-top my-3"
          />
          <div
            className="pb-2 pt-2 row"
          >
            <div
              className="col-md-4"
            >
              <label
                className="form-label"
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 14,
                  }
                }
              >
                Date on cheque:
              </label>
            </div>
            <div
              className="col-md-8"
            >
              <div
                className="row"
              >
                <div
                  className="col-md-4"
                >
                  <input
                    className="form-control"
                    placeholder="Month"
                    type="text"
                  />
                </div>
                <div
                  className="col-md-3"
                >
                  <input
                    className="form-control"
                    placeholder="Day"
                    type="text"
                  />
                </div>
                <div
                  className="col-md-3"
                >
                  <input
                    className="form-control"
                    placeholder="Year"
                    type="text"
                  />
                </div>
                <div
                  className="col-md-2"
                >
                  <div
                    style={
                      Object {
                        "border": "1px dotted grey",
                        "height": 25,
                        "marginRight": 10,
                        "textAlign": "center",
                        "width": 20,
                      }
                    }
                  >
                    P
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="border-top my-3"
          />
        </div>
        <div>
          <h5
            className="pb-2 pt-2 mb-2 mt-3"
            style={
              Object {
                "borderBottom": "1px solid grey",
              }
            }
          >
            Step 3: Choose an account to pay the stop payment fee
          </h5>
          <p
            className="mt-2"
          >
            Select the account from which to pay the fee.
          </p>
          <div
            className="pb-2 pt-2 mt-2 row"
          >
            <div
              className="col-md-4"
            >
              <label
                className="form-label"
                style={
                  Object {
                    "color": "grey",
                    "fontSize": 14,
                  }
                }
              >
                Service fee from account:
              </label>
            </div>
            <div
              className="col-md-6"
            >
              <select
                className="form-select"
              >
                <option>
                  Choose an account
                </option>
              </select>
            </div>
          </div>
        </div>
        <div>
          <h5
            className="pb-2 pt-2 mb-2 mt-3"
            style={
              Object {
                "borderBottom": "1px solid grey",
              }
            }
          >
            Step 4: Review the details and confirm that you want to proceed
          </h5>
          <p>
            Read the following Terms and Conditions and select "I agree" to request stop payment.
          </p>
          <div
            className="mt-3 row"
          >
            <div
              className="col-md-12"
            >
              <div
                className="p-4"
                style={
                  Object {
                    "border": "1px solid grey",
                  }
                }
              >
                <h4>
                  Stop Payment Terms and Conditions
                </h4>
                <p>
                  (Not applicable to stop payments on pre-authorized payments set up using ZippyX Advantage Debit Card
                  <sup>
                    TM1
                  </sup>
                  details
                  <sup>
                    1
                  </sup>
                  )
                </p>
                <ul>
                  <li>
                    ZippyX will attempt to stop payment of the cheque or pre-authorized debit described below("item") and will require a reasonable period to process your stop payment order ("order").
                  </li>
                  <li>
                    If your order cannot be processed before the item is paid to the payee, we will refund the service charge and have not further liability to you. An order received after 6 pm ET will be considered to be received the next banking day.
                  </li>
                  <li>
                    Your order is in effect for 12 months from the date it is submitted, or the date of the item whichever is later.
                  </li>
                </ul>
              </div>
              <div
                className="p-4"
                style={
                  Object {
                    "background": "#a1a1a1",
                    "color": "white",
                  }
                }
              >
                <div
                  className="form-check"
                >
                  <input
                    className="form-check-input"
                    disabled={false}
                    id="agree-terms-conditions"
                    name="agree-terms-conditions"
                    type="checkbox"
                  />
                  <label
                    className="form-check-label"
                    htmlFor="agree-terms-conditions"
                    title=""
                  >
                    I agree to these terms and conditions and wish to request a stop payment.
                  </label>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="mt-4"
          style={
            Object {
              "borderBottom": "1px solid grey",
              "borderTop": "1px solid grey",
            }
          }
        >
          <div
            className="mt-4 mb-4 row"
          >
            <div
              className="col"
            >
              <button
                className="d-flex btn btn-light"
                disabled={false}
                type="button"
              >
                <div
                  style={
                    Object {
                      "border": "1px dotted grey",
                      "height": 25,
                      "marginRight": 10,
                      "textAlign": "center",
                      "width": 20,
                    }
                  }
                >
                  P
                </div>
                Cancel
              </button>
            </div>
            <div
              className="d-flex justify-content-end col"
            >
              <button
                className="zippy-btn d-flex btn btn-primary"
                disabled={false}
                type="button"
              >
                Next
              </button>
            </div>
          </div>
        </div>
        <div
          className="mt-2 pt-2"
        >
          <div
            className="d-flex"
          >
            <div
              style={
                Object {
                  "border": "1px dotted grey",
                  "height": 25,
                  "marginRight": 10,
                  "textAlign": "center",
                  "width": 20,
                }
              }
            >
              P
            </div>
            <span
              style={
                Object {
                  "fontWeight": "bold",
                }
              }
            >
              Note:
            </span>
          </div>
          <p
            className="mt-2"
          >
            <b>
              Service Fee Details
            </b>
          </p>
          <p>
            A stop payment fee of $12.50 will be charged when you provide full details.
          </p>
          <p>
            A stop payment fee of $20 will be charged when you don't provide full details
          </p>
          <p>
            Full details on a cheque are Cheque Number and Amount.
          </p>
          <p>
            Full details on a pre-authorized payment are Payee and Amount.
          </p>
          <p>
            This fee doesn't apply to ZippyX Smart
            <sup>
              TM
            </sup>
            Plus Accounts or Wood Gundy Asset Advantage Accounts.
          </p>
          <p>
            <sup>
              1
            </sup>
            For stop payments on pre-authorized debits or recurring payments set up with ZippyX Advantage Debit Card details, contact the merchant, biller or payee directly and tell them to stop charging your account through your ZippyX Advantage Debit Card. If they don't stop, ZippyX may be able to work with Visa* to request a refund for you.
          </p>
          <p>
            <sup>
              TM 1
            </sup>
            Trandemark of ZippyX.
          </p>
          <p>
            <sup>
              *
            </sup>
            Trandemark of Visa Int., used under license
          </p>
        </div>
      </div>
      <div
        className="col-md-3"
      >
        <div
          className="p-3 bg-white mt-2 card"
        >
          <div
            className="card-title h5"
          >
            You can also:
          </div>
          <div
            className="card-body"
          >
            <a
              className="text-decoration-none text-black"
              href="/"
              style={
                Object {
                  "display": "block",
                  "fontSize": 12,
                  "marginBottom": 5,
                  "marginTop": 5,
                }
              }
            >
              Pay Bills
              <span
                style={
                  Object {
                    "color": "rgb(220, 53, 69)",
                    "marginLeft": 5,
                  }
                }
              >
                &gt;
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="footer-content-wrapper"
  >
    <div
      className="quick-links container"
    >
      <div
        className="row"
      >
        <h3
          className="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-2"
        >
          <ul>
            <li>
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                className="text-decoration-none"
                href="/legal"
                onClick={[Function]}
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          className="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      className="container mb-3"
    >
      <div
        className="d-none d-md-block"
      >
        <div
          className="d-flex justify-content-between border-bottom"
        />
        <div
          className="d-flex justify-content-between border-top"
        >
          <div
            className=""
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            className="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        className="d-block d-md-none"
      >
        <div
          className="d-flex border-bottom"
        >
          <div
            className="d-flex flex-column me-2"
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          className="d-flex justify-content-end border-top"
        >
          <div
            className="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
